wavesrv/pkg/remote/openai/openai.go

```diff
@@ -68,6 +74,96 @@ func RunCompletion(ctx context.Context, opts *sstore.OpenAIOptsType, prompt []ss
 	return marshalResponse(apiResp), nil
 }
 
+func RunCloudCompletion(ctx context.Context, opts *sstore.OpenAIOptsType, prompt []sstore.OpenAIPromptMessageType) ([]*packet.OpenAIPacketType, error) {
+	if opts == nil {
+		return nil, fmt.Errorf("no openai opts found")
+	}
+	if opts.Model == "" {
+		return nil, fmt.Errorf("no openai model specified")
+	}
+	cloudCompletionRequestConfig := sstore.OpenAICloudCompletionRequest{
+		Model:      opts.Model,
```

Comment:
currently sends model to the server - we shouldn't do this if we don't plan to use it

Reply:
agree, we should hard code the model on the server side

Reply:
addressed